manticore/ethereum/crytic_compile_compat.py

```diff
@@ -0,0 +1,189 @@
+"""
+Compatibility wrapper for crytic-compile 0.3.x to work with Manticore's expected 0.2.x API
```

Comment:
Can we drop this compatibility layer and use the newer crytic-compile API directly? see e.g. https://github.com/elopez/manticore/commit/b61de69b5e866dc591ea41056e0d2154260a6cad